src/main/java/org/minions/devfund/chelo/MovieDatabase.java

```diff
@@ -0,0 +1,186 @@
+package org.minions.devfund.chelo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * MovieDatabase class.
+ */
+public class MovieDatabase {
+    private ArrayList<Movie> movieList;
+    private ArrayList<Actor> actorList;
+
+    /**
+     * MovieDatabase constructor.
+     */
+    MovieDatabase() {
+        movieList = new ArrayList<>();
+        actorList =  new ArrayList<>();
+    }
+
+    /**
+     * Getter for MovieList.
+     * @return movie list
+     */
+    public ArrayList<Movie> getMovieList() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)